### DISCUSSION TITLE
How can i use osm test data

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#8051] [Q&A] [answered]
Hi there !

I want to know, how can I use test data of osm for organic maps with an android emulator.

I work on this issue https://github.com/organicmaps/organicmaps/issues/2376 and that can be usefull for me ^^.

Thanks in advance

 Arthur-GYT

**Comment by biodranik** [comment, marked as answer]
You can check already merged PRs with similar added tags, like for social networks or drive-through.

**Reply by ghost** [reply]
I don't understand can you explain ^^' ? The problem for me it's than I need om data for the map with a restaurant with this tag in the case of the issue.
